starter-code/styles/modules.css

```diff
@@ -9,12 +9,23 @@
   padding: 0 0 .2em 1em;
 }
 
-.icon-menu {
-  /* TODO: Add styles for the icon-menu class (whether it be the
-      hamburger font icon or an image of your choosing). It should be
-      bigger, and off to the right. */
+@media (max-width: 960px) {
```

Comment:
Media queries should always utilize min-width. You should design your page with a small screen in mind (mobile-first), and then, when the screen hits a **minimum width**,  those styles inside that media query are used, and they remain at any size above that width (or until another media query with a larger min width is triggered).